lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/stack/StackPresenter.java

```diff
@@ -423,12 +423,13 @@ private void mergeLeftButton(TopBarOptions options, TopBarButtons buttons, View
         }
     }
 
-    private void mergeBackButton(TopBarButtons buttons) {
+    private void mergeBackButton(TopBarButtons buttons, StackController stack) {
         if (buttons.back.hasValue() && isNullOrEmpty(buttons.left)) {
             if (buttons.back.visible.isFalse()) {
                 topBar.clearLeftButtons();
             } else {
-                topBar.setBackButton(createButtonController(buttons.back));
+                if (stack.size() > 1)
```

Comment:
Either make this a single-line conditional or surround the statement with curly brackets